test-support/helpers/ember-tooltips.js

```diff
@@ -123,7 +139,7 @@ export function triggerTooltipTargetEvent($element, type, options={}) {
   let wasEventTriggered = false;
 
   if (options.selector) {
-    $element = $element.find(options.selector);
+    $element = getTooltipTargetFromBody(options.selector);
```

Comment:
This change and other similar changes were necessary as a part of this PR because tooltips are no longer guaranteed to be children of the elements we were previously finding tooltips under. For example, in this case `$element` did not contain many of the tooltips we were looking for.